packages/launcher-component/src/loaders/buildimage-loader.tsx

```diff
@@ -25,25 +26,11 @@ interface BuildImageSuggestionsLoaderProps {
 }
 
 const convertToPairs = object => {
-  const result: Array<{ key: string; value: string }> = [];
-  let i = 0;
-  for (const k in object) {
-    if (object.hasOwnProperty(k)) {
-      result[i++] = {key: k, value: object[k]};
-    }
-  }
-  if(result.length === 0) {
-    return [{ key: '', value: ''}];
-  }
-  return result;
+  return (_.toPairs(object) || ['', '']) as string[][];
```

Comment:
@edewit did you check this?

Reply:
This is not needed at all, as it also handled inside the component:
```suggestion
  return _.toPairs(object);
```